src/main/java/com/google/sps/servlets/Table.java

```diff
@@ -0,0 +1,26 @@
+package com.google.sps.servlets;
+ 
+import java.util.ArrayList;
+import java.util.List;
+
+public class Table {
```

Comment:
Table can also be an Autovalue, and you probably need it with a builder . https://github.com/google/auto/blob/master/value/userguide/builders.md

Reply:
Doing this in later PR.